monitor/grafana/zeebe.json

```diff
@@ -192,22 +192,24 @@
         {
           "aliasColors": {},
           "bars": false,
+          "cacheTimeout": null,
           "dashLength": 10,
           "dashes": false,
-          "datasource": "$DS_PROMETHEUS",
```

Comment:
Not sure I get this change? Every panel should have the data source pointing to the `$DS_PROMETHEUS` variable/query, that way the dashboard is fully portable for other users to import in their own Grafana instances.

Reply:
:man_shrugging: :sweat_smile: I just exporting it :laughing:

Reply:
I think you have to set the panel to use `$DS_PROMETHEUS` as data source and it will be there.